test/index.ts

```diff
@@ -27,6 +27,8 @@ import * as pfy from '@google-cloud/promisify';
 import * as sinon from 'sinon';
 import * as spnr from '../src';
 
+const apiConfig = require('../src/spanner_grpc_config.json');
```

Comment:
Can you `import` this instead?

Reply:
It seems in TS it shows error "[ts] Cannot find module './spanner_grpc_config.json'."

Reply:
then ignore me :)